web/api/alert.go

```diff
@@ -16,12 +16,19 @@ package api
 import (
 	"fmt"
 	"net/http"
+	"encoding/json"
 
 	"github.com/julienschmidt/httprouter"
 
 	"github.com/prometheus/alertmanager/manager"
 )
 
+// Return all currently firing alerts as JSON
+func (s AlertManagerService) getAlerts(w http.ResponseWriter, r *http.Request, p httprouter.Params) {
+	aggs := s.Manager.GetAll(nil)
+	json.NewEncoder(w).Encode(aggs)
```

Comment:
Could you do:

```
respondJSON(w, aggs)
```

This handles error cases (failure to marshal JSON) correctly and is consistent with the API methods in `silence.go`.